server/app/models/grid_service.rb

```diff
@@ -13,7 +13,7 @@ class GridService
   field :stateful, type: Boolean, default: false
   field :user, type: String
   field :container_count, type: Integer, default: 1
-  field :cmd, type: Array
+  field :cmd, type: Array, default: []
```

Comment:
This might bite... not sure what is the best way to handle?

Reply:
Looks like it changes the `:cmd => []` that the agent sees via RPC... but it seems like both the agent `weavewait` wrapper logic and the Docker API ignores that empty array, and still uses the image's `CMD`.